x-pack/solutions/observability/plugins/slo/server/services/update_slo.ts

```diff
@@ -71,18 +72,15 @@ export class UpdateSLO {
     );
 
     const rollbackOperations = [];
+
     await this.repository.update(updatedSlo);
```

Comment:
why not add repository update in parallelisation as well? 
on the route we are already checking if user has permission to SLO write, not much can go wrong here i imagine.